src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,64 @@
-export const Tabs = () => {};
+import { FC, useState } from 'react';
+import classNames from 'classnames';
+import { Tab } from './types/interface';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void;
+}
+
+export const Tabs: FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  let currentTab = tabs.find(({ id }) => id === selectedTabId);
+
+  if (!currentTab) {
+    const [tab] = tabs;
+
+    currentTab = tab;
+  }
+
+  const [text, setText] = useState(currentTab.content);
```

Comment:
The `Tabs` component should be stateless, but you are using the `useState` hook to manage the text state. Instead, you should use the `selectedTabId` to determine the content of the selected tab.